traffic_ops/v4-client/servercheck.go

```diff
@@ -41,3 +43,23 @@ func (to *Session) GetServersChecks() ([]tc.GenericServerCheck, tc.Alerts, ReqIn
 	reqInf, err := to.get(API_SERVERCHECK, nil, &response)
 	return response.Response, response.Alerts, reqInf, err
 }
+
+// GetServerCheckWithParamHdr fetches the Delivery Service with the given ID.
+func (to *Session) GetServerCheckWithParamHdr(params url.Values, header http.Header) (*tc.GenericServerCheck, tc.Alerts, ReqInf, error) {
```

Comment:
Can we just name this "GetServerChecks"? If something already exists that doesn't use url.Values and a header we can just get rid of it and replace it with this one that does, since this is the new client.

Reply:
Done and updated all tests (within serverchecks_test.go) accordingly.